src/lcm/entry_point.py

```diff
@@ -48,7 +48,7 @@ def get_lcm_function(model, targets="solve", interpolation_options=None):
     # ==================================================================================
     # preparations
     # ==================================================================================
-    if targets not in {"solve", "simulate"}:
+    if targets not in {"solve", "simulate", "solve_and_simulate"}:
```

Comment:
Did not look thoroughly, but is this being tested?

Reply:
I've added an additional test to check that `solve_and_simulate` produces the same result as first calling `solve` and then calling `simulate`.